extension/src/popup/components/sendPayment/SendConfirm/SubmitResult/index.tsx

```diff
@@ -104,6 +107,29 @@ export const SubmitFail = () => {
         );
       }
     }
+
+    if (isPathPayment) {
+      const resultCodes = get(error, "response.extras.result_codes.operations");
+      if (resultCodes.includes("op_under_dest_min")) {
```

Comment:
can you add this to the `RESULT_CODES` enum? extension/src/popup/helpers/parseTransaction.ts

Reply:
ah I looked around and didn't see that 🤦 , ya  I'll add